MdeModulePkg/Universal/Variable/RuntimeDxe/Variable.c

```diff
@@ -3203,8 +3203,8 @@ ReclaimForOS (
 {
   EFI_STATUS      Status;
   UINTN           RemainingCommonRuntimeVariableSpace;
-  UINTN           RemainingHwErrVariableSpace;
-  STATIC BOOLEAN  Reclaimed;
+  UINTN           RemainingHwErrVariableSpace; // 2405_REBASE Will Conflict with 0a38ef7 
+  static BOOLEAN  Reclaimed; // MU_CHANGE: Use lowercase static for static lifetime
 

```

Comment:
Why is this specifically using lower case?

Reply:
Because this was the only place in the codebase that used a static lifetime variable (as opposed to private to a function) that gets compiled in a unit test. With the change to remove STATIC in GoogleTest, we had to preserve this one and it was the approach that Sean preferred :).